README.md

```diff
@@ -89,16 +89,26 @@ This installation guide is for ISIS users interested in installing ISIS (3.6.0)+
 
 7.  Finally, setup the environment variables:
 
+    For versions of ISIS 4.2.0 and earlier, use: 
+
```

Comment:
Soooo nitpicky, but should this get moved below line 94 and 95 since those lines are true for both cases?

Reply:
Yeah! Thanks for catching this. I made a slight different edit than the one you suggested, but I think it still solves the problem?